tests/test_views.py

```diff
@@ -44,4 +42,3 @@ def test_post_detail_response_with_correct_template(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "blog/post_detail.html")
```

Comment:
There is a typo in the template name. Use 'blog/post-detail.html' instead of 'blog/post_detail.html' in the `assertTemplateUsed` method.